README.md

```diff
@@ -41,10 +41,10 @@ docker run -v $(pwd):/home/data -i -t gwas_qc_pipe snakemake -k --use-conda -npr
 # Installing GwasQcPipeline on ccad2
 - Install miniconda and then create GwasQcPipeline production environment
 ```
-$ mkdir /scratch/myfolder/GwasQcPipeline_v1.2
-$ cd /scratch/myfolder/GwasQcPipeline_v1.2
+$ mkdir /scratch/myfolder/GwasQcPipeline_v1.4.0
+$ cd /scratch/myfolder/GwasQcPipeline_v1.4.0
 $ wget https://repo.anaconda.com/miniconda/Miniconda3-py39_4.12.0-Linux-x86_64.sh
```

Comment:
We could simply use `wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh` to ensure the user always gets the latest version of Miniconda, and so we don't have to update this ever so often.

Reply:
Good suggestion. Is there any concern that a later version of Miniconda will present any software version compatibility issues?

Reply:
Valid point - but I don't think there is any concern. Miniconda itself is just managing the environment and other software packages. And all the dependencies are specified in the `pyproject.toml` file so don't see how it would cause any compatibility issues. With that said, we can talk about it this morning with the group.

Reply:
Discussed in group meeting. General consensus was it likely won't cause any issues, but for now will leave as is for the specified Miniconda version and potentially later look into whether latest versions are fine.